.github/workflows/ticdc_integration_cron.yaml

```diff
@@ -1,8 +1,17 @@
 name: CDC Integration Tests Cron
 
+# TODO: revert this change
 on:
-  schedule:
-    - cron: '0 17-23 * * *' # run at minute 0 every hour from 01:00 ~ 07:00 UTC+8
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'dm/**'
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'dm/**'
```

Comment:
Is this change used for test here?

Reply:
> When the code review is over, I will revert the trigger mechanism for the corn job.

I find the comment.

Reply:
Yes!

Reply:
see: https://github.com/pingcap/ticdc/pull/3204#issuecomment-955715717